torch/utils/data/__init__.py

```diff
@@ -2,8 +2,17 @@
     (Sampler, SequentialSampler, RandomSampler,
      SubsetRandomSampler, WeightedRandomSampler, BatchSampler)
 from torch.utils.data.dataset import \
-    (Dataset, IterableDataset, TensorDataset, ConcatDataset, ChainDataset,
-     Subset, random_split, Dataset as MapDataPipe, IterableDataset as IterDataPipe)
+    (ChainDataset,
+     ConcatDataset, 
+     DataChunk as DataChunk,
+     Dataset, 
+     Dataset as MapDataPipe,
+     DFIterDataPipe as DFIterDataPipe,
+     IterableDataset,
+     IterableDataset as IterDataPipe,
```

Comment:
nit: why `import IterableDataset` and `import IterableDataset as IterDataPipe`?

Reply:
cleaned